frontend/src/utils/Pane.svelte

```diff
@@ -10,117 +10,31 @@
     flex-flow: row;
     justify-content: space-between;
     align-items: stretch;
+    overflow: hidden;
   }
 
   .inner {
-    overflow: auto;
+    overflow: hidden;
     flex-grow: 1;
   }
-
-  .minimap-container {
-    background: var(--main-bg-color);
-    display: flex;
-    z-index: 1;
-  }
-
-  .minimap {
-    width: 64px;
-    max-width: 64px;
-    display: flex;
-    flex-flow: column;
-    justify-content: space-between;
-    align-items: center;
-  }
 </style>
 
 <script>
-  import { onMount } from "svelte";
-
-  export let paddingVertical = "3em",
-    paddingHorizontal = "3em",
-    scrollY = undefined,
-    displayMinimap = false;
-
+  export let paddingVertical = "0",
```

Comment:
Reflecting on our discussion, I think we should have the default be at least `1em`, maybe `0.5em` at the very very least. Then, we can override this value to be `0` for the tree view since that's where it will look the best to have the pane move vertically.

My thinking is that this will make the default for non-tree views in the top left area of the page look much better. For example, there should (in my opinion) be more padding below the buttons at the bottom here:

<img width="902" alt="image" src="https://github.com/redballoonsecurity/ofrak/assets/99368685/3009d731-d6a8-4f2a-bbf4-e953dc3ccd35">